storage/innobase/xtrabackup/doc/source/installation/docker.rst

```diff
@@ -1,46 +1,102 @@
-template for newly created containers. `Docker` images for Percona XtraBackup
-are hosted publicly on Docker Hub `here <https://hub.docker.com/r/percona/percona-xtrabackup>`__.
+A `Docker` container is based on a `Docker` image, which works as a
+template for newly created containers. The `Docker` images for Percona XtraBackup are hosted publicly on the ` Docker Hub <https://hub.docker.com/r/percona/percona-xtrabackup>`__.
+
+Install `Docker`
+================================================================================
+
+The Docker engine can be installed on many platforms. Consult `Install Docker Engine <https://docs.docker.com/engine/install/>`__ for the latest versions, system requirements, and instructions.
+
+Create and Run a Percona XtraBackup Docker container
+===============================================================================
+
+1. Create a directory for the container. Within that directory, pull the latest image of Percona XtraBackup from Docker Hub repository. This operation may take a few minutes. 
+
+   .. code-block:: bash
+
+      $ docker pull percona/percona-xtrabackup
+
+2. After the download is complete, create the container in an interactive mode. The ``-i`` option attaches the standard input stream (stdin) of the bash shell and the ``-t`` creates a terminal for the process. 
+
+   The hostname is automatically generated when you create a container. For this example, the hostname is ``660cdc388af`` in these steps. The hostname for your container will be different.
+
+   .. code-block:: bash
+
+      $ docker run -it percona/percona-xtrabackup /bin/bash
+      [root@e660cdc388af /]#
+
+3. You access the container as root. 
+
+   .. code-block:: bash
+
+      [root@e660cdc388af /]# xtrabackup --version
+      xtrabackup version 8.0.26-18 based on MySQL server 8.0.26 Linux (x86_64) (revision id: 4aecf82)
+
+Run a Container in daemon mode
+==================================================
+
+You can run a container in detached (daemon) mode to run in the background. 
 
 .. code-block:: bash
 
-   $ sudo docker create ... percona/percona-xtrabackup --name xtrabackup ...
+   $ docker run -it -d percona/percona-xtrabackup bash
 
-.. rubric:: Scope of this section
+You can verify that the container is running with the following command:
 
-`Docker` containers offer a range of different options effectively allowing
-to create quite complex setup. This section demonstrates how to backup data
-on a Percona Server for MySQL running in another `Docker` container.
+.. code-block:: bash
 
-Installing `Docker`
-================================================================================
+   $ docker ps 
+   CONTAINER ID   IMAGE                        COMMAND   CREATED              STATUS              PORTS     NAMES
+   ffb00224ea9f   percona/percona-xtrabackup   "bash"    30 seconds ago       Up 29 seconds                 sleepy_maxwell
+
+To access the container, use the following command, which uses the container ID. 
```

Comment:
use the following command that uses the container ID.

Reply:
Rewritten